### PR TITLE
Fixed it

### PR DESCRIPTION
**Pull request by Kaszz**
## Description
Fixed the issue by instantly publishing if connection is established to a machine that is already subscribed to.

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes, if necessary.
- [x] All tests pass (existing and potential new).
- [ ] Sonar check has passed
- [x] No codesmells or bugs

### CODE REVIEW

**Review by Wylardt — APPROVED** [review]

**Review by Lawst96 — APPROVED** [review]
Not an optimal fix, but it'll do.

**Comment by Kechr** [comment]
Er vi blevet ligeglade med vores coverage?

**Comment by Kaszz** [comment]
> Er vi blevet ligeglade med vores coverage?

Jeg tror ikke vi har nogen idé om hvordan man skal teste et publish statement.

**Review by Wylardt — APPROVED** [review]

File: BrewMesModulesParent/livedata/src/test/java/com/brewmes/livedata/LiveDataControllerTest.java [hunk 1]
```diff
@@ -17,15 +19,31 @@
...
 
+    @Mock
+    ILiveDataService liveDataService;
+
     @InjectMocks
     LiveDataController liveDataController;
 
     @Test
-    void liveDataTest() {
+    void liveDataTest_true() {
```
Comment (Lawst96) [inline BrewMesModulesParent/livedata/src/test/java/com/brewmes/livedata/LiveDataControllerTest.java:29]:
```suggestion
    void liveDataTest_noData() {
```

File: BrewMesModulesParent/livedata/src/test/java/com/brewmes/livedata/LiveDataControllerTest.java [hunk 2]
```diff
@@ -17,15 +19,31 @@
...
         when(subscribeService.subscribeToMachineValues(ID)).thenReturn(true);
         liveDataController.liveData(ID);
         verify(subscribeService, times(1)).subscribeToMachineValues(ID);
     }
 
+    @Test
+    void liveDataTest_null() {
```
Comment (Lawst96) [inline BrewMesModulesParent/livedata/src/test/java/com/brewmes/livedata/LiveDataControllerTest.java:36]:
```suggestion
    void liveDataTest_dataIsPresent() {
```

**Review by Lawst96 — APPROVED** [review]

**Review by Kechr — APPROVED** [review]

**Pull request merged**